build/tutorials/exchanges/exchange-c-chain-integration.md

```diff
@@ -0,0 +1,70 @@
+"log-level": "verbo",
+"log-dir": "./log/"
+}
+```
+
+### Interacting with the C-Chain
+
+Interacting with the C-Chain is identical as interacting with `go-ethereum`. You can find the reference material for C-Chain API [here](../../avalanchego-apis/contract-chain-c-chain-api.md).
+
+Please note that `personal_` namespace is turned off by default. To turn it on, you need to pass the appropriate command line switch to your node, like in the above config example.
+
+## Constructing transactions
+
+Avalanche C-Chain transactions are identical to standard EVM transactions with 2 exceptions:
+* They must be signed with Avalanche’s ChainID (43114).
+* The gas price is fixed to 225 Gwei.
+
+For development purposes, Avalanche supports all the popular tooling for Ethereum, so developers familiar with Ethereum and Solidity can feel right at home. We have tutorials and repositories for several popular development environments:
+* [MetaMask and Remix](../../tutorials/smart-contracts/deploy-a-smart-contract-on-avalanche-using-remix-and-metamask.md)
+* [Truffle](../../tutorials/smart-contracts/using-truffle-with-the-avalanche-c-chain.md)
+* [Hardhat](https://github.com/ava-labs/avalanche-smart-contract-quickstart)
+
+## Ingesting On-Chain Data
+
+You can use any standard way of ingesting on-chain data you use for Ethereum network.
+
+### Determining Finality
+
+Avalanche consensus provides fast and irreversible finality with 1-2 seconds. To query the most up to date finalized block, query any value (i.e. block, balance, state, etc) with the `latest` parameter.  If you query above the last finalized block (i.e. eth_blockNumber returns 10 and you query 11), an error will be thrown indicating that unfinalized data cannot be queried (as of avalanchego@v1.3.2).
+
+### (Optional) Custom Golang SDK
+
+If you plan on extracting data from the C-Chain into your own systems using golang, we recommend using our custom [ethclient](https://github.com/ava-labs/coreth/tree/master/ethclient). The standard go-ethereum Ethereum client does not compute block hashes correctly (when you call `block.Hash()`) because it doesn't take into account the added `[ExtDataHash](https://github.com/ava-labs/coreth/blob/2c3cfac5f766ce5f32a2eddc43451bdb473b84f1/core/types/block.go#L98)` header field in Avalanche C-Chain blocks, which is used move AVAX between chains (X-Chain and P-Chain). You can read more about our multi-chain abstraction [here](../../../learn/platform-overview/README.md) (out of scope for a normal C-Chain integration).
+
+If you plan on reading JSON responses directly or use web3.js (doesn't recompute hash received over the wire) to extract on-chain transaction data/logs/receipts, you shouldn't have any issues!
+
+## Summary
+
+We hope that his document will enable you to quickly integrate with Avalance Network on the C-Chain. If you have any problems or questions, reach out to us either directly to our developers, or on our public [Discord](https://chat.avalabs.org/) server.
```

Comment:
I'd remove this: `We hope that his document will enable you to quickly integrate with Avalance Network on the C-Chain.`